### ISSUE TITLE
fixedDecimalLength doesn't appear to work?

### ISSUE THREAD

**Issue by ianstormtaylor** [cchanxzy/react-currency-input-field#117]
```tsx
<CurrencyInput
  defaultValue={0}
  fixedDecimalLength={2}
/>
```

The input will render with `0` instead of `0.00`.

**Comment by cchanxzy** [comment]
Oops I've never noticed that! Thanks for raising this issue.

It should be fixed now with a PR I've just merged.

Also, I think you should use `decimalScale` instead of `fixedDecimalLength`. `fixedDecimalLength` kinda does something else 😄

```js
<CurrencyInput
  defaultValue={0}
  decimalScale={2}
/>
```

**Comment by ianstormtaylor** [comment]
An interesting, I didn't realize it wasn't the right prop, thank you. The latest release with `decimalScale={2}` doesn't seem to work still. It only applies the fixed decimal length after the input is blurred, but not when it is first rendered?

```jsx
<CurrencyInput
  decimalScale={2}
  defaultValue={0.1}
/>
```

![image](https://user-images.githubusercontent.com/311752/103976827-c97df300-5145-11eb-8e73-0f472e200f45.png)


Here's a sandbox: https://codesandbox.io/s/inspiring-goldberg-5c3oy?file=/src/App.js

**Comment by cchanxzy** [comment]
Ah yes you're right again!

I will add a fix soon.

**Comment by cchanxzy** [comment]
This should be fixed now in latest release 3.0.2 *fingers crossed*

Feel free to re-open this issue, or submit a new issue if you find another bug.

Once again, thanks for raising the bug!
